[Trivia]: Michael Jackson & Prince Lyrics

Aurorum opened the pull request:
### Type

- [ ] Bugfix
- [x] Enhancement (part of #12)
- [ ] New feature

### Description of the changes

I thought it'd be fun to offer lyrics as a trivia, but it can be tricky selecting songs which are known widely enough that a significant number of people have a reasonable chance of answering the questions correctly.

As such, to narrow down the scope, these add trivia questions for Michael Jackson and Prince, two of the most well-known artists across the world whose musical collections are extensive and popular enough to dedicate an entire trivia too.

They're comprised of a variety of lyrics, two lyrics for each song. Players need to name the song from the lyric. Most of the songs are well-known songs from the artist, but there are plenty of challenges too.

Aurorum committed:
Add Prince Lyrics Trivia

Aurorum committed:
Add Michael Jackson Trivia

Aurorum committed:
Double the number of Prince lyrics

DevilXD commented:
This is missing a changelog entry. Please see here: https://github.com/Cog-Creators/Red-DiscordBot/blob/V3/develop/.github/CONTRIBUTING.md#47-using-towncrier

Aurorum committed:
Create 12.enhance.rst

Aurorum commented:
Added a changelog entry, thanks 👍

Aurorum committed:
Merge branch 'V3/develop' into add/lyrics-trivia

Aurorum committed:
Merge branch 'V3/develop' into add/lyrics-trivia

Flame442 commented on redbot/cogs/trivia/data/lists/michaeljackson.yaml:
```diff
@@ -0,0 +1,113 @@
+- Dirty Diana
+(Identify the Michael Jackson song) Fever...temperatures rising now:
+- Don't Stop 'Til You Get Enough
+- Don't Stop Till You Get Enough
```
You might want to consider adding an answer that removes the apostrophe in `Don't`.

Flame442 commented on redbot/cogs/trivia/data/lists/michaeljackson.yaml:
```diff
@@ -0,0 +1,113 @@
+(Identify the Michael Jackson song) I see the kids in the street with not enough to eat, who am I to be blind pretending not to see their needs?:
+- Man in the Mirror
+(Identify the Michael Jackson song) At night when the stars shine, I pray in you I'll find a love so true:
+- I Just Can't Stop Loving You
```
You might want to consider adding an answer that removes the apostrophe in `Can't.`

Flame442 commented on redbot/cogs/trivia/data/lists/michaeljackson.yaml:
```diff
@@ -0,0 +1,113 @@
+(Identify the Michael Jackson song) Please have mercy cause I just can't take this:
+- Scream
+(Identify the Michael Jackson song) Black man, blackmail, throw the brother in jail:
+- They Don't Care About Us
```
You might want to consider adding an answer that removes the apostrophe in `Don't`.

Flame442 commented on redbot/cogs/trivia/data/lists/michaeljackson.yaml:
```diff
@@ -0,0 +1,113 @@
+- This Time Around
+(Identify the Michael Jackson song) They thought they really could control me:
+- This Time Around
+(Identify the Michael Jackson song) Did you ever stop to notice all the children dead from war? Did you ever stop to notice this crying Earth's weeping shore?:
```
Might be "This crying earth, **this weeping shore**".

Aurorum replied:
I checked the lyrics booklet and it's "the" - I've heard it wrong my whole life!

Flame442 commented on redbot/cogs/trivia/data/lists/michaeljackson.yaml:
```diff
@@ -0,0 +1,113 @@
+- You Are Not Alone
+(Identify the Michael Jackson song) Speculate to break the one you hate, circulate the lie you confiscate:
+- Tabloid Junkie
+(Identify the Michael Jackson song) They say he's homosexual:
```
This question *might* not suitable for core red due to the content. Needs discussion.

Flame442 commented on redbot/cogs/trivia/data/lists/michaeljackson.yaml:
```diff
@@ -0,0 +1,113 @@
+- Earth Song
+(Identify the Michael Jackson song) What have we done to the world? Look what we've done. What about all the peace that you pledged your only son?:
+- Earth Song
+(Identify the Michael Jackson song) I bet he never had a social life anyway, you think he brother with the KKK?:
```
This question *might* not suitable for core red due to the content. Needs discussion.

Flame442 commented on redbot/cogs/trivia/data/lists/michaeljackson.yaml:
```diff
@@ -0,0 +1,113 @@
+- D.S
+- D.S.
+(Identify the Michael Jackson song) Tom Sneddon is a cold man:
+- D.S
```
You might want to consider adding an answer that removes the period (`D S` and `DS`).

Flame442 commented on redbot/cogs/trivia/data/lists/prince.yaml:
```diff
@@ -0,0 +1,221 @@
+(Identify the Prince song) If I was your best friend, would you let me take care of you:
+- If I Was Your Girlfriend
+(Identify the Prince song) I go out every night and sleep all day since you took your love away:
+- Nothing Compares 2 U
```
You might want to consider adding an answer `Nothing compares to you`.

Flame442 commented on redbot/cogs/trivia/data/lists/prince.yaml:
```diff
@@ -0,0 +1,221 @@
+(Identify the Prince song) A journal full of numbers that I used to go through, all behind me now:
+- BREAKDOWN
+(Identify the Prince song) So here I sit in my lonely room, looking for my sunshine:
+- 17 Days
```
You might want to consider adding an answer `Seventeen days`.

Flame442 commented on redbot/cogs/trivia/data/lists/prince.yaml:
```diff
@@ -0,0 +1,221 @@
+(Identify the Prince song) So let the rain come down, let the rain come down, let the rain come down down:
+- 17 Days
+(Identify the Prince song) Every time I comb my hair, thoughts of you get in my eyes:
+- Erotic City
```
This song *might* not suitable for core red due to the content. Needs discussion.

Aurorum replied:
I just replaced the questionable songs with some better suited one, I'm not adament to have them included in the trivia. :)

Flame442 commented on redbot/cogs/trivia/data/lists/prince.yaml:
```diff
@@ -0,0 +1,221 @@
+(Identify the Prince song) There will come a time where love will blow your mind and everything you look for, you'll find:
+- Diamonds And Pearls
+(Identify the Prince song) There will be a new city with the streets of gold, the young so educated they'll never grow old:
+- 7
```
You might want to consider adding an answer `Seven`.

Flame442 commented on redbot/cogs/trivia/data/lists/prince.yaml:
```diff
@@ -0,0 +1,221 @@
+(Identify the Prince song) Rain sounds so cool when it hits the barn roof and the horses wonder who you are:
+- Raspberry Beret
+(Identify the Prince song) So what if we're controlling all the oil, is it worth a child dying for?:
+- Money Don't Matter 2 Night
```
You might want to consider adding answers for `tonight` and `dont`.

Flame442 commented on redbot/cogs/trivia/data/lists/prince.yaml:
```diff
@@ -0,0 +1,221 @@
+(Identify the Prince song) Together with love for one another, we shall make them known:
+- All The Midnights In The World
+(Identify the Prince song) I can't disguise the pounding of my heart, it beats so strong:
+- Take Me With U
```
You might want to consider adding an answer for `Take me with you`.

Flame442 commented on redbot/cogs/trivia/data/lists/prince.yaml:
```diff
@@ -0,0 +1,221 @@
+- My Name Is Prince
+(Identify the Prince song) The sky was all purple, there were people running everywhere trying to run from the destruction:
+- 1999
+(Identify the Prince song) Everybody's got a bomb, we could all die anyday:
```
Probably should be `any day`.

Flame442 commented on redbot/cogs/trivia/data/lists/prince.yaml:
```diff
@@ -0,0 +1,221 @@
+- The Beautiful Ones
+(Identify the Prince song) You were so hard to find, the beautiful ones, they hurt you every time:
+- The Beautiful Ones
+(Identify the Prince song) She took me to her castle and I just couldn't believe my eyes:
```
This song *might* not suitable for core red due to the content. Needs discussion.

Aurorum replied:
Removed :+1:

Flame442 commented on redbot/cogs/trivia/data/lists/prince.yaml:
```diff
@@ -0,0 +1,221 @@
+(Identify the Prince song) Loneliness already knows you:
+- Around The World In A Day
+(Identify the Prince song) Was it good for you, was that what you wanted me to be?:
+- Controversey
```
Misspelled, should be `Controversy`.

Flame442 commented on redbot/cogs/trivia/data/lists/prince.yaml:
```diff
@@ -0,0 +1,221 @@
+(Identify the Prince song) Let's groove, September, Earth Wind and Fire:
+- Musicology
+(Identify the Prince song) Electric word life, it means forever and that's a mighty long time:
+- Let's Go Crazy
```
You might want to consider adding an answer for `Lets go crazy`.

Flame442 commented on redbot/cogs/trivia/data/lists/prince.yaml:
```diff
@@ -0,0 +1,221 @@
+- Let's Go Crazy
+(Identify the Prince song) Better live now before the grim reaper comes knocking on your door:
+- Let's Go Crazy
+(Identify the Prince song) Don't you miss the feeling music gave ya back in the day?:
```
Repeated from lines 164-167.

Flame442 commented on redbot/cogs/trivia/data/lists/prince.yaml:
```diff
@@ -0,0 +1,221 @@
+- 3 Chains o Gold
+- 3 Chains Of Gold
+(Identify the Prince song) I'm not a woman, I'm not a man, I am something that you'll never understand:
+- I Would Die 4 U
```
You might want to consider adding an answer for `I would die for you`.

Flame442 commented on redbot/cogs/trivia/data/lists/prince.yaml:
```diff
@@ -0,0 +1,221 @@
+(Identify the Prince song) I'm not your lover, I'm not your friend, I am something that you'll never comprehend:
+- I Would Die 4 U
+(Identify the Prince song) My luck will change tonight, there's gotta be a better life:
+- Baby I'm A Star
```
You might want to consider adding an answer for `Baby Im a star`.

Flame442 reviewed (changes requested):
There's a few questions that have answers that could be misleading that might need some extra answers since the bot mostly only looks for exact matches. There's also a few songs that have some content that I do not believe is suitable for core red, however I think more discussion is needed so I am marking this PR as such.

Aurorum committed:
Update Prince trivia

Aurorum committed:
Update Michael Jackson trivia

Aurorum committed:
Merge branch 'V3/develop' into add/lyrics-trivia

Aurorum commented:
Thanks @Flame442! :)

I've addressed the review now by replacing some of the questionable songs/lyrics with songs/lyrics that should be fine to include, adding some different options for answers, and fixing the typos you mentioned.

Aurorum committed:
Missed one

Flame442 reviewed (approved):
All set!

Aurorum committed:
Merge branch 'V3/develop' into add/lyrics-trivia

Tobotimus committed:
Merge branch 'V3/develop' into add/lyrics-trivia
